test/test_va_api_fixture.cpp

```diff
@@ -37,6 +38,7 @@
 #include <unistd.h> // for close()
 #include <va/va_drm.h>
 #endif
+#include <xf86drm.h>
```

Comment:
Seems like the header should be within the `#else` block just above. Not sure how I missed that earlier.